tensorflow/python/ops/metrics_impl.py

```diff
@@ -2067,6 +2067,115 @@ def compute_recall(true_p, false_n, name):
     return rec, update_op
 
 
+@tf_export('metrics.sensitivity')
+def sensitivity(labels,
+                predictions,
+                weights=None,
+                metrics_collections=None,
+                updates_collections=None,
+                name=None):
+  """Same as `tensorflow.metrics.recall`
+
+  For additional information about specificity and sensitivity, see the
+  following: https://en.wikipedia.org/wiki/Sensitivity_and_specificity"""
+  return recall(labels, predictions, weights, metrics_collections,
+                updates_collections, name or "sensitivity")
+
+@tf_export('metrics.specificity')
+def specificity(labels,
```

Comment:
Hi @zasdfgbnm,
I think what Josh suggesting is keep your old way but change the default name. (Josh to verify)
it will be something like
def specifity(..., name='specifity'): 
  return recall(..., name)

Reply:
@ispirmustafa `specificity` is not recall, nor anything else already in tensorflow. `sensitivity` is the same as recall. Changing the default name can be either by
```python
def sensitivity(..., name=None):
  return recall(..., name or "sensitivity")
```
or by
```python
def sensitivity(..., name="sensitivity"):
  return recall(..., name)
```
The difference between these two cases is the generated api doc. See:
https://www.tensorflow.org/api_docs/python/tf/metrics/recall
it has:
```python
tf.metrics.recall(
    labels,
    predictions,
    weights=None,
    metrics_collections=None,
    updates_collections=None,
    name=None
)
```
So I would prefer the first approach because having a  `name="sensitivity"` for sensitivity and a `name=None` for everything else seems to be confusing to users.

Reply:
I'm fine with this.